app/controllers/ballots_controller.rb

```diff
@@ -1,18 +1,43 @@
 class BallotsController < ApplicationController
-  before_action :load_beer, only: [:create]
+  before_filter :load_beer, only: [:create]
+  before_filter :load_ballot, only: [:show, :edit, :update, :destroy]
+  before_filter :load_all_beers, only: [:new, :edit]
```

Comment:
^^ This could all be replaced with cancancan. Dont need to fix it for this PR, but if oyu are looking for ways to improve the code. That would be one way to do it